### ISSUE TITLE
Filter and number of results was not accessible - button added

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#347]
Please review the addition of the Update Results button that Vino added in Test 3. This was necessary for Accessibility.

**Comment by AlOneill** [comment]
A little confused about what exactly we are being asked to test — do not remember what was there previously. The "Update results" button appears to work OK.

I did notice that the close button (the X) in the "When you searched for" box is not accessible by keyboard (using tab key).

The "Tools" drop-down needs a solid white background — the horizontal rule appears behind the first item.

Also, the table does not have a caption — it does not need to be visible, but should be there.

I note that "Filter by" and "Show" do not work with the Individual or Record Type sort (on the column headings) — that will need some careful explanation in the Help!

**Comment by DeniseColbert** [comment]
Previously it was just the dropdown boxes and selection was automatic which was flagged as a problem for accessibility (I think keyboard users?). Vino changed it to require the button-press after selection.

**Comment by DeniseColbert** [comment]
I've put you back on the story @Vino-S to take a look at Alison's feedback/make stories (although positive review of the button)

**Comment by richpomfret** [comment]
@Vino-S to check and see that these are done.

**Comment by Vino-S** [comment]
_A little confused about what exactly we are being asked to test — do not remember what was there previously. The "Update results" button appears to work OK._
Thanks @AlOneill 

_I did notice that the close button (the X) in the "When you searched for" box is not accessible by keyboard (using tab key)._

Fixed it in test3.

_The "Tools" drop-down needs a solid white background — the horizontal rule appears behind the first item._

Updated and ready for test in test3

_Also, the table does not have a caption — it does not need to be visible, but should be there._
Added caption and ready for test in test3

_I note that "Filter by" and "Show" do not work with the Individual or Record Type sort (on the column headings) — that will need some careful explanation in the Help!_
Intresting bug. will create a new issue

**Comment by AlOneill** [comment]
@Vino-S Close button has visible focus — all good.

Tools drop-down background looks good.
Why is is button styled to be the inverse of other buttons? Not sure how confusing that is to users, but it is puzzling. @lychee616 what do you think?

Caption now present (thank you) but we can make it more helpful: suggest adding "Column 1 has link to details of record; column 2 can be sorted on given name; column 3 can be sorted by event."
@lychee616 What do you think would be useful for a screen-reader user to hear? I am just making it up as I go!

**Comment by Vino-S** [comment]
Ready for testing in test3

**Comment by richpomfret** [comment]
@AlOneill to test.

**Comment by AlOneill** [comment]
Looking good! Thank you @Vino-S
